src/main/java/org/kpmp/repositoryDataset/RepositoryFileDataset.java

```diff
@@ -229,12 +231,42 @@ public void setTissueSource(String tissueSource) {
 
     @Nullable
     @JsonProperty("release_version")
-    public Integer getReleaseVersion(){
+    public String getReleaseVersion(){
         return releaseVersion;
     }
 
-    public void setReleaseVersion(Integer releaseVersion){
+    public void setReleaseVersion(String releaseVersion){
         this.releaseVersion = releaseVersion;
     }
 
-}
\ No newline at end of file
+    @Transient
+    private String participantIdSort;
```

Comment:
Let's keep all of our variables defined at the top of the class...having them interspersed in the file makes the code much less readable.